src/server/status/index.js

```diff
@@ -53,6 +56,9 @@ export default function (kbnServer, server, config) {
   server.route(wrapAuth({
     method: 'GET',
     path: '/status',
+    config: {
+      tags: ['api']
```

Comment:
This may be due to the dev proxy.  Can you run Kibana with `npm start -- --no-base-path` and see the behaviors at that point?